tests/test-cluster.py

```diff
@@ -463,3 +464,119 @@ def test_no_cert_reissue_in_nodes(self):
         for vm in self.VM:
             lock_files = vm.run("ls /var/snap/microk8s/current/var/lock/")
             assert "no-cert-reissue" in lock_files.decode()
+
+
+class TestUpgradeCluster(object):
+    @pytest.fixture(autouse=True, scope="module")
+    def setup_old_versioned_cluster(self):
+        """
+        Provision VMs of the previous version and form the cluster.
+        """
+        try:
+            print("Setting up cluster of an older version")
+            track, risk_level = channel_to_test.split('/')
```

Comment:
What would the track be if we are on strict on eksd?

Reply:
Do we want to mix those clusters, the eksd or strict with the normal one?

Reply:
normally we would not need to.

Reply:
So, these tracks will be mixed with the older version of themselves. Like 1.24-eksd with 1.23-eksd and so on.